site/loading.js

```diff
@@ -77,7 +77,10 @@ function getOrigin () {
 
 var apiServer = getOrigin()
 
-var displayName = window.location.pathname.replace('/repo', '').slice(1)
+var path = window.location.pathname.replace('/repo', '').slice(1)
+var splitted = path.split('/')
+var displayName = splitted.slice(0, 2).join('/')
+var deepLink = '/' + splitted.slice(2).join('/')
```

Comment:
Results in an extra `/` if there is no deep path, which I guess won't matter, but this might be a bit more clear?

```
var deepLink = splitted.length > 2 ? '/' + splitted.slice(2).join('/') : ''
```